src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -198,8 +207,18 @@ public void setReadOnlyAuthToken(String token) {
          * if this value is not specified.
          */
         public String getWriteAuthToken() {
```

Comment:
I suggest a simpler approach for testing this. Instead of adding @ExposedOnlyForTesting setter, we can just create a new package-protected method like:

```
String getWriteAuthToken(EnvVariables envVariables)
//or:
String getWriteAuthToken(String envVarValue)
```

Then, we can make the public ```getWriteAuthToken()``` delegate to the new package-protected method. We would only test package-protected method.

I don't think we necessarily have to use "ExposedForTesting" for package protected methods - they are not really exposed. They are package-private! :)